content/concept-stand-alone-mode.md

```diff
@@ -0,0 +1,103 @@
+
+If you only want the most recent ledger, you can skip this step.
+
+If you want to load a specific historical ledger version, use the [`ledger_request` command](reference-rippled.html#ledger-request) to make `rippled` fetch it. If `rippled` does not already have the ledger version, you may have to run the `ledger_request` command multiple times until it has finished retrieving the ledger.
+
+If you want to replay a specific historical ledger version, you must fetch both the ledger version to replay and the ledger version immediately before it.
+
+### 4. Shut down `rippled`. ###
+
+Use the [`stop` command](reference-rippled.html#stop):
+
+```
+rippled stop --conf=/path/to/rippled.cfg
+```
+
+### 5. Start `rippled` in stand-alone mode. ###
+
+To load the most recent ledger version, you can use the `-a` and `--load` options when starting the server:
+
+```
+rippled -a --load --conf=/path/to/rippled.cfg
+```
+
+To instead load a specific historical ledger, use the `--load` parameter along with the `--ledger` parameter, providing the ledger index or identifying hash of the ledger version to load:
+
+```
+rippled -a --load --ledger 19860944 --conf=/path/to/rippled.cfg
+```
+
+### 6. Manually advance the ledger. ###
+
+When you load a ledger with `--ledger` in stand-alone mode, it goes to the current open ledger, so you must [manually advance the ledger](#advancing-ledgers-in-stand-alone-mode):
+
+```
+rippled ledger_accept --conf=/path/to/rippled.cfg
+```
+
+
+Advancing Ledgers in Stand-Alone Mode
+-------------------------------------------------------------------------------
+
+In stand-alone mode, `rippled` does not communicate to other members of the peer-to-peer network or participate in a consensus process. Instead, you must manually advance the ledger index using the [`ledger_accept` command](reference-rippled.html#ledger-accept):
+
+```
+rippled ledger_accept --conf=/path/to/rippled.cfg
+```
+
+In stand-alone mode, `rippled` makes no distinction between a "closed" ledger version and a "validated" ledger version. (For more information about the difference, see [The Ripple Ledger Consensus Process](https://ripple.com/knowledge_center/the-ripple-ledger-consensus-process/).)
+
+Whenever `rippled` closes a ledger, it reorders the transactions according to a deterministic but hard-to-game algorithm. (This is an important part of consensus, since transactions may arrive at different parts of the network in different order.) When using `rippled` in stand-alone mode, you should manually advance the ledger before submitting a transaction that depends on the result of a previous transaction. Otherwise, the second transaction might be executed before the first transaction when you manually advance the ledger. **Note:** You can safely submit multiple transactions from a single address to a single ledger, because `rippled` sorts transactions from the same address in ascending order by `Sequence` number.
```

Comment:
Linked "Sequence number" to https://ripple.com/build/transactions/#common-fields -- and it's "highlighted" because it's a field name in JSON, which we always typeset in monospace font.

Rephrased the part about advancing the ledger as follows:
> When using `rippled` in stand-alone mode, you should manually advance the ledger before submitting a transaction that depends on the result of a transaction from a different address. Otherwise, the two transactions might be executed in reverse order when the ledger is closed.